numpy/distutils/command/build.py

```diff
@@ -36,6 +38,16 @@ def initialize_options(self):
         self.cpu_baseline = "min"
         self.cpu_dispatch = "max -xop -fma4" # drop AMD legacy features by default
         self.disable_optimization = False
+        """
+        the '_simd' module is a very large. Adding more dispatched features
```

Comment:
@mattip, @rgommers,
During the last rework, I took into account the size over the performance which lead to massively reduce the size, comparing with the old patch in #13516.
```C
$ strip -g _simd.cpython-36m-x86_64-linux-gnu.so
$ du _simd.cpython-36m-x86_64-linux-gnu.so
1656	_simd.cpython-36m-x86_64-linux-gnu.so
$ gzip _simd.cpython-36m-x86_64-linux-gnu.so
$ du _simd.cpython-36m-x86_64-linux-gnu.so.gz
236	_simd.cpython-36m-x86_64-linux-gnu.so.gz
```
still, large comparing with the other test modules, especially we still in the beginning, and a large number of intrinsics are expected to be added within the upcoming pull-requests.